microsoft-365/managed-desktop/get-started/register-devices-partner.md

```diff
@@ -17,69 +17,41 @@ This topic describes the steps for Partners to follow to register devices. The p
+Before completing registration for a customer, you must first establish a relationship with them at the [Partner Center] (https://partner.microsoft.com/dashboard). See the [consent documentation] (https://docs.microsoft.com/windows/deployment/windows-autopilot/registration-auth#csp-authorization) for more details on that process. Any CSP partner can add devices on behalf of any customer, as long as the customer consents. You can also learn more about partner relationship and Autopilot permissions at [Partner Center help] (https://docs.microsoft.com/partner-center/customers_revoke_admin_privileges#windows-autopilot)
 
-To complete registration for your customer, first create a CSV file.
 
 >[!NOTE]
->For your convenience, you can download a [sample CSV file](https://github.com/MicrosoftDocs/microsoft-365-docs/raw/public/microsoft-365/managed-desktop/get-started/downloads/device-registration-sample-partner.csv) for this *Partner version*.
+> This documentation is only for Partners and OEMs. The process for self-registration is documented in [Register devices in Microsoft Managed Desktop yourself](register-devices-self.md).
 
-Your file needs to include the **exact same column headings** as the sample one (Manufacturer, Model, etc.), but your own data for the other rows. If you use the template, open it in a text editing tool such as Notepad, and consider leaving all the data in row 1 alone, only entering data in rows 2 and below. 
-    
-  ```
- Manufacturer,Model,Serial Number
-  SpiralOrbit,ContosoABC,000000000000
-  
-  
-  ```
 
+## Register devices by using Partner Center
 
+Once you have established the relationship with your customers, you can leverage Partner Center to add devices to Autopilot for any of the customers that you have a relationship with following the below steps :
 
+1. Navigate to [Partner Center](https://partner.microsoft.com/dashboard)
+2. Select **Customers** from the Partner Center menu and then select the customer whose devices you want to manage.
+3. On the customer's detail page, select **Devices**.
+4. Under **Apply profiles** to devices select **Add devices**.
+5. Enter **Microsoft365Managed_Autopilot** for the Group Name and then select **Browse** to upload the customer's list (in .csv file format) to Partner Center.
 
->[!NOTE]
->This format is only for the Partner process. The process for self-registration is documented in [Register devices in Microsoft Managed Desktop yourself](register-devices-self.md).
 
 >[!IMPORTANT]
->These values must match the manufacturer values from SMBIOS exactly, including capitalization and special characters. 
+>The Group Name must match exactly **Microsoft365Managed_Autopilot** including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
```

Comment:
```suggestion
> The Group Name must match **Microsoft365Managed_Autopilot** exactly, including capitalization and special characters. This will allow the newly registered devices to be assigned with the Microsoft Managed Desktop Autopilot profile.
```